numpy/ctypeslib.pyi

```diff
@@ -1,8 +0,0 @@
-from typing import Any
-
-load_library: Any
-ndpointer: Any
-ctypes_load_library: Any
-c_intp: Any
-as_ctypes: Any
-as_array: Any
```

Comment:
To be clear, only `ctypes_load_library` is deprecated.

Reply:
Ah, my bad. Everything should be in order as of https://github.com/numpy/numpy/pull/17104/commits/2ec7afa3f4fcc5cb8792ac9f63ef38f667506c5a.